frontend/src/app/style-dialog/style-dialog.component.html

```diff
@@ -39,4 +39,20 @@ <h3>Trace: {{traceName}}</h3>
 
 <!-- Change colors -->
 <h3>Color Options:</h3>
-<p>TODO</p>
+<div class="picker">
+    <div class="circle" style="background: rgb(0,0,0)" (click)="changeColor(0,0,0)"></div>
+    <div class="circle" style="background: rgb(31, 119, 180)" (click)="changeColor(31, 119, 180)"></div>
```

Comment:
suggestion - Is it possible to define color names in html and then use the names instead of rgb?

Reply:
There is a list of colors that I could specify by name but none of them are the colors I use here. The colors used here are the same as the default plotly colors they use for consistency.